### ISSUE TITLE
Clear ElasticSearch index on Staging/Production to re-index posts for author autocomplete

### ISSUE THREAD

**Issue by rclee91** [Seneca-CDOT/telescope#3332]
After #3301 gets merged, posts will need to be re-indexed for the author autocomplete query in `Search` to work. 
The current `Search` query shouldn't be affected either way, but the new change would be nice to have. 

@manekenpix if you could please clear any cache involved with ElasticSearch indexing on staging/production when the release happens, that would be awesome.
This would likely include Redis cache and any stored ElasticSearch index data.

**Comment by manekenpix** [comment]
Elasticsearch data cleared in both `staging` and `production`.
